packages/admin/cms-admin/src/generator/future/generateFormField.ts

```diff
@@ -26,9 +27,7 @@ export function generateFormField(
     if (!introspectionField) throw new Error(`didn't find field ${name} in gql introspection type ${gqlType}`);
     const introspectionFieldType = introspectionField.type.kind === "NON_NULL" ? introspectionField.type.ofType : introspectionField.type;
 
-    const requiredByIntrospection = introspectionField.type.kind == "NON_NULL";
-
-    const required = config.required ?? requiredByIntrospection; //if undefined default to requiredByIntrospection
+    const required = !isFieldOptional({ config, gqlIntrospection, gqlType });
```

Comment:
We need to decide if we still want to support `config.required` before merging this.

Reply:
alright, for me it seems this is decided to keep the support, because @thomasdax98 said there are several use-cases for this.

Reply:
We'll discuss this in tomorrow's meeting.